test/tests/interfacekernels/1d_interface/no-failed-point-inversions.i

```diff
@@ -0,0 +1,139 @@
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 1
+    nx = 10
+    xmax = 2
+  []
+  [subdomain1]
+    input = gen
+    type = SubdomainBoundingBoxGenerator
+    bottom_left = '1.0 0 0'
+    block_id = 1
+    top_right = '2.0 1.0 0'
+  []
+  [interface]
+    input = subdomain1
+    type = SideSetsBetweenSubdomainsGenerator
+    primary_block = '0'
+    paired_block = '1'
+    new_boundary = 'primary0_interface'
+  []
+  [break]
+    type = BreakMeshByBlockGenerator
+    input = interface
+  []
+  displacements = 'disp_x'
+[]
+
+[AuxVariables]
+  [disp_x][]
+[]
+
+[ICs]
+  [right]
+    type = ConstantIC
+    variable = disp_x
+    block = 1
+    value = 1
+  []
+[]
+
+[Variables]
+  [u]
+    order = FIRST
+    family = LAGRANGE
+    block = '0'
+  []
+
+
```

Comment:
Do not need many spaces

Reply:
Ok I removed the space 😄 Can you re-approve?